src/gromacs/nblib/atoms.h

```diff
@@ -45,45 +45,44 @@
 #define GROMACS_ATOMS_H
 
 #include <string>
-#include <tuple>
-#include <unordered_map>
-#include <vector>
 
 #include "gromacs/math/vectypes.h"
 
-#include "interactions.h"
-
-class TopologyBuilder;
-
 namespace nblib
 {
+using AtomKindName = std::string;
+using AtomMass = real;
+using AtomCharge = real;
+using C6Param = real;
+using C12Param = real;
 
 class Atom {
 public:
+
     Atom() noexcept;
 
-    Atom(std::string atomName,
-             real mass,
-             real charge,
-             real c6,
-             real c12);
+    explicit Atom(AtomKindName atomName,
```

Comment:
The explicit is not needed.

Reply:
Yes. Will remove. It doesn't tell the compiler not to implicitly deduce type.